### ISSUE TITLE
Suggestion - Rewarding Actifit Content

### ISSUE THREAD

**Issue by steemflow** [actifit/actifit#14]
Adding more values means , putting more effort on writing a short content too through the app. Like Daily Diary Entry, A Freewrite or U-Log Actifit app is currentlly awards the user who makes 5000 + Activity counts, I would suggest, why not put additional reward to those who write a short content with images even if they score less in Activity Counts. By this way, the activity count app can also act as a Blogging app. And helps in increasing the user base

**Comment by mcfarhat** [comment]
Thank you for reaching out.
Our reward already takes that into consideration. The reward value considers activity count, post content, media, upvotes, comments, user's actifit rank, moderator review

**Comment by steemflow** [comment]
By allowing this...any user with less than 5000 also get rewards. I believe currently those user are not getting rewards

**Comment by mcfarhat** [comment]
yes users with less than 5K do not get rewarded. We want the first criteria to be activity based, as that's the core value behind actifit :)
